yaks-rails/lib/yaks-rails.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails'
+require 'yaks-html'
+
+module Rails
+  module Yaks
+    def self.global_config
+      @global_config ||= ::Yaks.new
+    end
+
+    class << self
+      attr_writer :global_config
+    end
+
+    def self.configure(&block)
+      @global_config = ::Yaks.new(&block)
+    end
```

Comment:
I'm fine with these three going into the main `Yaks` namespace (so inside the `yaks` gem), so you would do `Yaks.configure` and not `Yaks::Rails.configure`. Fact of the matter is that for most use cases a single global configuration is fine, so might as well standardize how to do that.